Default.sublime-commands

```diff
@@ -1,38 +1,4 @@
 [
-    {
-        "caption": "Preferences: SublimeLinter Settings",
```

Comment:
Actually, it’s kind of a convention to have this and the one for keybindings. Lots of packages provide them and it makes them easy to access by keyboard.

Reply:
What about keybindings?

Reply:
Put both back in with 2668f20.

Reply:
Now that issue has been addressed, can you close that change request, because it has become a distraction by now. Thanks.